tests/crashes/123809.rs

```diff
@@ -1,4 +0,0 @@
-//@ known-bug: #123809
-type Positive = std::pat::pattern_type!(std::pat is 0..);
```

Comment:
I don't think keeping this test around is terribly useful, it just tests that a completely unfinished feature doesn't ICE if used on stable. But to even get here you get various unstable feature errors

Reply:
Seems reasonable.